src/states/abstract/State.js

```diff
@@ -86,6 +86,13 @@ export default class State {
     return false;
   }
 
+  /*
+   * Overriden by BeforePrerollState, PrerollState and ContentPlaybackState.
```

Comment:
Don't need `State` in the names

Reply:
Ah, I was following the convention elsewhere in this file

Reply:
It's probably my bad, I renamed them at one point during development. I think we should use the current file names.